app/models/menu_item.rb

```diff
@@ -0,0 +1,10 @@
+class MenuItem < ApplicationRecord
+  has_and_belongs_to_many :orders
+  belongs_to :menu
+
+  TYPES_OF_DISHES = [:first_course, :second_course, :drink]
+  enum menu_type: TYPES_OF_DISHES
```

Comment:
please use string-based enums
```
  enum limit_type: {
    committed: COMMITTED = 'committed'.freeze, 
    non_committed: NON_COMMITTED = 'non_committed'.freeze
  }, _prefix: true

```
also Rozhenko made a gem for this, but you better ask him whether it works fine.
also, why? because it won't create you any troubles to modify/rearrange/expand this enum https://sipsandbits.com/2018/04/30/using-database-native-enums-with-rails/